### DISCUSSION TITLE
eslint-mdx stuck when start to lint mdx files.

### DISCUSSION THREAD

**Discussion by Jeurei** [mdx-js/mdx#2121] [Q&A]
Hello. I'm want to install mdx eslint plugin on my nx monorepo.
I have @typescript-eslint/parser as default parser. I my ui-storybook/.eslintrc.json, I have add to overrides.
{
      "files": "*.mdx",
      "parser": "eslint-mdx"
}.
And after this eslint just stucking. In debug mode I see. 
<img width="1162" alt="изображение" src="https://user-images.githubusercontent.com/56633996/187908594-3e91c1e7-0a55-4c4a-9e69-8297531b68da.png">

**Comment by wooorm** [comment]
This is not enough information. Please follow our support guide and amend your question: https://mdxjs.com/community/support/.
